packages/ag-charts-community/src/chart/series/seriesTypes.ts

```diff
@@ -45,6 +45,7 @@ export interface SeriesNodeDatum {
     readonly itemId?: any;
     readonly datum: any;
     readonly point?: Readonly<SizedPoint>;
+    readonly valid?: boolean;
```

Comment:
nit: [isNA](https://en.wikipedia.org/wiki/N/A) might be a more suitable name to indicate that this is a missing data point.